workspace/src/test/kotlin/com/cosmotech/workspace/service/WorkspaceServiceImplTests.kt

```diff
@@ -159,8 +152,6 @@ class WorkspaceServiceImplTests {
         workspaceServiceImpl.createWorkspaceFile(ORGANIZATION_ID, WORKSPACE_ID, file, false, null)
     assertNotNull(workspaceFile.fileName)
     assertEquals("my_file.txt", workspaceFile.fileName)
-    assertTrue(
```

Comment:
All the "assertTrue/False" on the blob existence have been removed. Do we want / can we check if the S3 blob is actually there instead? Not sure if this is easy or a good idea though.

Reply:
I removed these checks because previously workspace files CRUD were done "directly on filesystem" and checks were possible in unit tests.
Right now, we use an external service, these checks should be covered by integrationTests (which is the case)